signature/mechanism_gpgme.go

```diff
@@ -133,12 +137,56 @@ func (m *gpgmeSigningMechanism) Sign(input []byte, keyIdentity string) ([]byte,
 	if err != nil {
 		return nil, err
 	}
+
+	passphraseFile := os.Getenv(PASSPHRASE_FILE_ENVIRONMENT)
```

Comment:
I still wish we didn’t add any environment variable like that to the mandatory code path.